source/write/bulk-write.txt

```diff
@@ -0,0 +1,334 @@
+.. _ruby-bulk-write:
+
+=====================
+Bulk Write Operations
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: insert, update, replace, code example, multiple changes
+
+Overview
+--------
+
+This guide shows you how to use the {+driver-short+} to perform a bulk
+write operation that makes multiple changes to your data in a single
+database call.
```

Comment:
Since most of the other standardized pages follow the pattern: "In this guide, you can learn...", I think we should start this the same way
```suggestion
In this guide, you can learn how to use the {+driver-short+} to perform a bulk
write operation that makes multiple changes to your data in a single
database call.
```